draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1763,54 +1773,71 @@ be carried in qlog data:
-utility of the dataset as a whole. As such, a balance should be found between
-logging these fields and the potential risks inherent in their (involuntary)
-disclosure. This balance depends on the use case at hand (e.g., research
-datasets might have different requirements to live operational troubleshooting).
-Capturing the minimal amount of data required for a specific purpose can help to
-minimize the risks associated with data usage. qlog implementations that provide
-fine-grained control over the inclusion of data fields, ideally on a
-per-use-case or per-connection basis, improve the ability to minimize data.
+## Operational implications and recommendations
 
-Any data that is determined to be necessary for a use case at hand could be
-logged or captured. As per {{!RFC6973}}, operators must be aware that such data
-will be at risk of compromise. As such, measures should be taken to firstly
-reduce the risk of compromise and secondly reduce the risk of abuse of
-compromised data. While a full discussion of both aspects is out of scope for
-this document, the following paragraphs discuss high-level considerations that
-can be applied to qlog data.
-
-To reduce the risk of compromise, operators can take measures such as: limiting
-the length of time that data is stored, encrypting data in transit and at rest,
-limiting access rights to the data, and auditing data usage practices. qlog
-deployments that provide integrated options for automated or manual data
-deletion and (aggressive) aggregation, improve the ability to minimize the risk
-of compromise.
-
-To reduce the risk of data abuse after compromise, data can be anonymized,
-pseudonymized, otherwise permutated/replaced, truncated, (re-)encrypted, or
-aggregated. A partial discussion of applicable techniques (especially for IP
-address information) can be found in {{Appendix B of !DNS-PRIVACY=RFC8932}}.
-Operators should, however, be aware that many of these techniques have been
-shown to be insufficient to safeguard user privacy and/or to protect user
-identity, especially if a qlog data set is large or easily correlated against
-other data sources.
+Operational implications qlog should focus on log capture and access.
+
+It is recommended that the ability to enable qlog capture is subject to access
+control and auditing. Furthermore, access control policy should extend to the
+level of information that is allowed to be captured (e.g., capture of
+more-sensitive data requires higher privileges).
+
+It is recommended that access to stored qlogs is subject to access control and
+auditing.
+
+Access control techniques in end user client environments can be limited.
+End-users have the possibility of capturing their own logs and risking their own
+security and privacy in ways that could be surprising. Implementations should
+consider how to make enabling qlog conspicous, and resistant to automation or
+drive-by attacks. Examples include, requiring explicit actions to start a
```

Comment:
... or social engineering?

Reply:
yep that was my implication but I'll add that